tfhe/c_api_tests/test_high_level_integers.c

```diff
@@ -64,6 +64,68 @@ int uint8_public_key(const ClientKey *client_key, const PublicKey *public_key) {
   return ok;
 }
 
+int uint8_safe_serialization(const ClientKey *client_key, const ServerKey* server_key) {
```

Comment:
I don't know if in the rust implementation there is a test, that checks that using 2 keys made with different (incompatible) params, if you mixmatch ciphertexts, the safe_deserialization properly returns that ciphertexts are not conformant ?

Reply:
Ok 👍